ftw/news/tests/test_news_portlets.py

```diff
@@ -111,15 +112,17 @@ def test_portlet_edit_form_cancel(self, browser):
         """
         This test makes sure that cancelling the editing of a portlet does
         not change the portlet.
+
+        TODO: Does this test really make any sense???
```

Comment:
@mbaechtold If I look to it now... no. Because the cancel hander is generic from plone.
But I remember there were problems with the old formlib implementation. 

You may delete it, or leave it there, your choice.

Reply:
I see... I leave the test but I have added a comment in the doc string.